### DISCUSSION TITLE
No transpose method

### DISCUSSION THREAD

**Discussion by mathmul** [spatie/simple-excel#99] [Ideas]
I wouldn't go as far as "transpose method is missing", and you might just quickly dismiss this request because it probably breaks the low-memory-usage rule, but just throwing it out there.

Some would probably find it helpful to be able to read transposed CSVs, ie. a CSV with rows as columns, so without a true header line, and optionally where the first element of every line is the header itself.

Example:
```csv
name,Juan,Thoo
age,20,21
gender,Male,Female
```
```php
$rows = SimpleExcelReader::create($pathToCsv)
    ->transpose()
    // ->noHeaderColumn() // Optionally instead of `->noHeaderRow()`
    ->getRows()
    ->each(function(array $columnProperties) {
       // in the first pass $columnProperties will contain
       // ['name' => 'Juan', 'age' => '20', 'gender' => 'Male']
});
```

**Comment by freekmurze** [comment]
Indeed, we'll probably not implement this and we want to focus on low-memory usage.
